src/applications/facility-locator/components/SearchControls.jsx

```diff
@@ -48,36 +102,29 @@ class SearchControls extends Component {
 
     return (
       <div className="search-controls-container clearfix">
-        <form>
+        <form onSubmit={this.handleSubmit}>
           <div className="columns usa-width-one-third medium-4">
-            <label htmlFor="streetCityStateZip" id="streetCityStateZip-label">Enter Street, City, State or Zip</label>
-            <input ref="searchField" name="streetCityStateZip" type="text" onChange={this.handleQueryChange} value={currentQuery.searchString} aria-label="Street, City, State or Zip" title="Street, City, State or Zip"/>
+            <label htmlFor="street-city-state-zip" id="street-city-state-zip-label">
+              Enter Street, City, State or Zip
+            </label>
+            <input
+              id="street-city-state-zip"
+              name="street-city-state-zip"
+              type="text"
+              onChange={this.handleQueryChange}
+              value={currentQuery.searchString}
+              aria-label="Street, City, State or Zip"
```

Comment:
Having a unique ID and a label with the `htmlFor={thatUniqueId}` gives us a 508 compliant label and means we can go without the `aria-label`. I like to use those in cases where the screen reader label is longer or more explicit than the visual label.